src/handlers/comment.ts

```diff
@@ -17,38 +18,50 @@ export async function handleComment(context: Context) {
     await addCommentToIssue(context, `@${sender}, You are not allowed to set access`, payload.issue.number);
   }
 
-  if (body.match(/\/.*/)) {
-    const { username, labels } = parseComment(body);
-    const { access, user } = context.adapters.supabase;
-    const url = payload.comment?.html_url as string;
-    if (!url) throw new Error("Comment url is undefined");
+  try {
+    if (/\/\S+/.test(body)) {
+      const { username, labels, command } = parseComment(body);
+      if (command === "/allow") {
```

Comment:
It looks pretty low level to be capturing commands like this. Do you think we should make a wrapper in the SDK so that it is more convenient for future plugin developers?

It's okay to leave it like this forever in this plugin, but I am keen on making this common capability easier for future developers.

Reply:
I added `commander-js` so at least the command parsing itself is robust, but I agree this might not be ideal. My main concern is commands colliding between different plugins. We could think of an SDK later on but in the end we would have to check for what command we called. Let me rework this part a little bit to make it look better.

Reply:
@0x4007 Here is a test result
https://github.com/Meniole/bot/issues/5#issuecomment-2112275828

Reply:
> My main concern is commands colliding between different plugins

This is why I wanted commands to be mappable from the config. But if not, then it will be the partners' responsibility to manage collisions. Perhaps collisions could be useful to double up functionality?